extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx

```diff
@@ -401,3 +453,24 @@ function _createStudyBrowserTabs(
 
   return tabs;
 }
+
+function _findTabAndStudyOfDisplaySet(displaySetInstanceUID, tabs) {
```

Comment:
You have `StudyInstanceUID` at time of save. Depending on how we order thumbnails, the newly created SR will either be the first or last SR in the list of thumbnails.

I think we can key most of this off either the current subscription, or identify a new platform level event that makes sense for what's occurring, and then accomplish all of this. Unfortunately, for scrollIntoView, each thumbnail may need a `ref`

Reply:
Kept generic for now as discussed.